sonarqube/datadog_checks/sonarqube/data/conf.yaml.example

```diff
@@ -154,6 +154,12 @@ instances:
     #
     # port: <PORT>
 
+    ## @param is_jmx - boolean - optional - default: false
+    ## Whether or not this instance is a configuration for a JMX integration.
+    ## If `is_jmx` is set to true at the init_config level, this flag is ignored.
```

Comment:
Maybe this one requires additional description that if it's true then other things are ignored too

Reply:
It's written in the options description https://github.com/DataDog/integrations-core/blob/master/sonarqube/datadog_checks/sonarqube/data/conf.yaml.example#L112, so i'm just adding a note